src/scala/scripts/ScalaPBGenerator.scala

```diff
@@ -38,8 +38,12 @@ class ScalaPBGenerator extends Processor {
     }
 
   def processRequest(args: java.util.List[String]) {
+    val curDir = Paths.get("").toAbsolutePath.toString
```

Comment:
I'm a bit out of context. Why do you need the currentDir? Mainly asking since directory locations get hairy with Bazel. @johnynek can probably elaborate more than me.

Reply:
So, I *think* this is okay since the rule is sandboxed and bazel won't let it read anything it is not supposed to read. I guess this is needed when you are depending on files that are not in subdirectories of the current file?

I'm a bit surprised why you don't get the full path passed in, however. Can you confirm that removing this actually shows an error in the tests?

Reply:
Sorry guys, I should roll back this particular change.

So the issue that I am having is regarding external proto imports. This is a separate issue, however, and should not be conflated with this. I'll create a separate issue and PR about that problem.